primitives/src/targeting/eval.rs

```diff
@@ -887,7 +895,7 @@ fn eval(input: &Input, output: &mut Output, rule: &Rule) -> Result<Option<Value>
                 .try_bignum()?;
             let deposit_asset = &input.global.channel.deposit_asset;
 
-            let divisor = get_deposit_asset_divisor(&deposit_asset)?;
+            let divisor = DEPOSIT_ASSETS_MAP.get(deposit_asset).unwrap();
```

Comment:
No unwraps, please. Check what error we should return in this case (TypeError?) or maybe just value `0` for the amount